oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/web/OAuth2AuthorizationRequestRedirectFilterTests.java

```diff
@@ -191,6 +192,8 @@ public void doFilterWhenNotAuthorizationRequestAndClientAuthorizationRequiredExc
 		request.setServletPath(requestUri);
 		MockHttpServletResponse response = new MockHttpServletResponse();
 		FilterChain filterChain = mock(FilterChain.class);
+		willAnswer((invocation) -> assertThat((invocation.<HttpServletResponse>getArgument(1)).isCommitted()).isFalse())
```

Comment:
Will you please move this assertion into its own test and then add a comment to the top of that test that looks like this:

```java
// gh-11602
```